AfricanReligions/events/afr_roog_religious_traditions.txt

```diff
@@ -0,0 +1,121 @@
+################################################
+#
+# Events for Religious Traditions
+#
+#################################################
+
+namespace = GROUPS
+
+long_character_event = { #Landing Event
+	id = GROUPS.1000
+	title = "GROUPSTITLE"
+	desc = "EVTDESCGROUPS1000"
+	picture = "GFX_evt_tribal_shrine"
+
+	is_triggered_only = yes
+
+	trigger = { NOT = { is_inaccessible_trigger = yes } }
+
+	option = {
+		name = "EVTOPTAGROUPS.1000"
+		ai_chance = {
+			factor = 1
+			modifier = {
+				has_education_diplomacy_trigger = yes
+				factor = 2
+			}
+			modifier = {
+				trait = gregarious
+				factor = 2
+			}
+		}
+		hidden_tooltip = { long_character_event = { id = GROUPS.1001 } }

```

Comment:
Instead of hiding this, I think there should be a custom tooltip that indicates to the player that clicking it will not be a permanent choice.  Something like "You will consider the Serer branch"  or something like that.